pkg/map/route.go

```diff
@@ -18,6 +25,10 @@ func RouteMapper(r []RouteGetter) *routeMapper {
 }
 
 func (m *routeMapper) Map(targetDnsName string, targetPort string) (*Route, error) {
```

Comment:
can you also make a test for this to check if giving kedge an ip addr returns an error?

Reply:
Done.